src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
```

Comment:
Add validation checks to ensure that the input data is valid before processing it. For example, check that the people array is not empty before processing it.